odl/tomo/geometry/parallel.py

```diff
@@ -343,76 +596,225 @@ class Parallel3dAxisGeometry(ParallelGeometry, AxisOrientedGeometry):
     ray direction.
 
     In the standard configuration, the rotation axis is ``(0, 0, 1)``,
-    the detector reference point starts at ``(1, 0, 0)``, and the
+    the detector reference point starts at ``(-1, 0, 0)``, and the
     initial detector axes are ``[(0, 1, 0), (0, 0, 1)]``.
     """
 
-    def __init__(self, apart, dpart, axis=[0, 0, 1], **kwargs):
+    _default_config = dict(axis=(0, 0, 1),
+                           det_pos_init=(-1, 0, 0),
+                           det_axes_init=((0, 1, 0), (0, 0, 1)))
+
+    def __init__(self, apart, dpart, axis=(0, 0, 1), **kwargs):
         """Initialize a new instance.
 
         Parameters
         ----------
         apart : 1-dim. `RectPartition`
-            Partition of the angle interval
+            Partition of the angle interval.
         dpart : 2-dim. `RectPartition`
-            Partition of the detector parameter interval
+            Partition of the detector parameter rectangle.
         axis : `array-like`, shape ``(3,)``, optional
-            Fixed rotation axis defined by a 3-element vector
-        det_init_pos : `array-like`, shape ``(3,)``, optional
-            Initial position of the detector reference point. The zero
-            vector is only allowed if ``det_init_axes`` is explicitly
-            given.
-            By default, a `perpendicular_vector` to ``axis`` is used.
-        det_init_axes : 2-tuple of `array-like`'s (shape ``(3,)``), optional
-            Initial axes defining the detector orientation.
-            By default, the normalized cross product of ``axis`` and
-            ``det_init_pos`` is used as first axis and ``axis`` as second.
+            Vector defining the fixed rotation axis of this geometry.
+        det_pos_init : `array-like`, shape ``(3,)``, optional
+            Initial position of the detector reference point.
+            The default depends on ``axis``, see Notes.
+        det_axes_init : 2-tuple of `array-like`'s (shape ``(3,)``), optional
+            Initial axes defining the detector orientation. The default
+            depends on ``axis``, see Notes.
+        extra_rot : `array_like`, shape ``(3, 3)``, optional
+            Rotation matrix that should be applied at the end to the
+            configuration of ``det_pos_init`` and ``det_axes_init``.
+            The rotation is extrinsic, i.e., defined in the "world"
+            coordinate system.
+
+        Notes
+        -----
+        In the default configuration, the rotation axis is ``(0, 0, 1)``,
```

Comment:
I'll probably add a pointer to the online doc, no space to add an in-depth explanation here.